docs/modules/ROOT/pages/documentation/payara-server/request-tracing-service/tracing-remote-ejbs.adoc

```diff
@@ -0,0 +1,149 @@
+[[tracing-remote-ejbs]]
+= Tracing Remote EJBs
+
```

Comment:
Please add the following under the heading as we usually do to specify the first version to support a feature:
```suggestion

`_Since Payara Server 5.2020.4_`

```